pages/instructorAssessmentInstance/instructorAssessmentInstance.sql

```diff
@@ -57,13 +57,22 @@ SELECT
     -- PostgreSQL so it can be properly interpreted when a grade
     -- update POST is received back.
     CAST(iq.modified_at AS TEXT) AS modified_at,
+    -- If auto_points and manual_points are null, these columns were
+    -- not updated since the implementation of separate manual and
+    -- auto points. This computation, in practice, assigns
+    -- manual_points if the question is manually graded or if the
+    -- points are above max_auto_points.
```

Comment:
Do we have a plan to clean this up in the DB? I'd be happy to run a migration that does this update and then we can eliminate this code. I'm worried that we have many copies of this code block (and comment) but in the future it will be easy to forget this.

Reply:
I can do that, I was just worried that this would be a pretty long migration script. If you're ok with running it (twice, once on canary migration and once on full deployment for the leftovers) I'm happy to create a new script.

Reply:
I think in this case it's worth it to run a long migration to avoid the permanent code complexity in many places. I think the code should assume that the migration has been run so we can simplify all these places where we have code like above.

Reply:
Done in d4d0321. I used the same migration file instead of a separate one because (1) both need to run again on deployment, and (2) the instance question update depends on the assessment question update, so they need to run in order.

Reply:
Looks good, thanks!